modules/graphql_image/graphql_image.info.yml

```diff
@@ -5,4 +5,5 @@ package: GraphQL
 core: 8.x
 dependencies:
   - image
+  - responsive_image
```

Comment:
You can't make the whole module depend on responsive_image because then, even if you are not using responsive images but just want the basic image functionality, you'd have to have the module enabled.